custom-vu/src/main/kotlin/jces1209/vu/page/DcIssuePage.kt

```diff
@@ -57,4 +63,66 @@ class DcIssuePage(
     override fun linkIssue(): DcIssueLinking {
         return DcIssueLinking(driver)
     }
+
+    override fun changeAssignee(): DcIssuePage {
+        driver
+            .findElement(By.id("assignee-val"))
+            .click();
+
+        val userPicker = driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.id("assignee-form"))
+            )
+
+        Actions(driver)
+            .sendKeys(Keys.DELETE, Keys.ARROW_DOWN)
+            .perform()
+
+        driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.id("assignee-group-suggested"))
+            )
+
+        var allUsers = driver
+            .wait(
+                ExpectedConditions.presenceOfAllElementsLocatedBy(By.xpath("//*[@id='all-users']//*[@role='option']"))
+            )
+
+        var user = if (allUsers.count() >= 2) {
+            allUsers[2].text.substringBefore(" - ")
```

Comment:
This can throw. `List.get` is indexed from 0.

- [ ] careful when accessing by index